wcfsetup/install/files/style/ui/imageViewer.scss

```diff
@@ -236,7 +236,7 @@ $wcfImageViewerFontColor: rgba(211, 211, 211, 1);
 			-ms-user-select: none;*/
 			
 			@extend .icon48;
-			@extend .icon-spinner;
+			@extend .fa-spinner;
```

Comment:
Without the changes in this file the new version crashes during compilation, but the SCSS was broken before anyway.